src/examples/forms/radio/Validation.tsx

```diff
@@ -31,8 +30,8 @@ const Example = () => {
   return (
     <Row justifyContent="center">
       <Col size="auto">
-        <MD isBold>Choose a growth type</MD>
-        <div role="group" aria-label="Choose a growth type">
+        <Fieldset>
```

Comment:
Should we care too hard that the version using the `Fieldset` shows the radios spaced at the (now) proper 8px margin, but all the others, where no `Fieldset` is present have them stacked with no margin? 

![image](https://user-images.githubusercontent.com/1230973/118051652-35f84100-b336-11eb-968e-880411908eb6.png)

Reply:
Yeah, I do care. But as I attempted to align all the examples I came to the belief that, if we decide to expose a treatment, then all the toggle-style examples need to be cut down to singles and we should have a separate new `<Fieldset>` page showing how various groupings work out. In other words, the situation needs design consideration.

Reply:
(also note that the change you're suggesting turns all of the existing semi-bold labels into regular weight)

Reply:
I think that’s fair (a new page with cross linking).